python/mujinplanningclient/realtimeitl3planningclient.py

```diff
@@ -65,13 +119,14 @@ def GetITLState(self, robotname=None, timeout=10, fireandforget=False, **kwargs)
         """
         taskparameters = {
             'command': 'GetITLState',
-        }
+        }  # type: Dict[str, Any]
         if robotname is not None:
             taskparameters['robotname'] = robotname
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout, fireandforget=fireandforget)
 
     def ExecuteTrajectory(self, identifier, trajectories, statevalues=None, stepping=False, istep=None, cycles=1, restorevalues=None, envclearance=15, robotspeed=None, robotaccelmult=None, timeout=10, fireandforget=False):
+        # type: (Any, Any, Any, Any, Any, Any, Any, float, Optional[float], Optional[float], float, bool) -> Any
```

Comment:
Please add a ToDo to determine these types properly

Reply:
Done